central/reports/scheduler/schedule.go

```diff
@@ -119,8 +119,13 @@ type ReportRequest struct {
 }
 
 type reportEmailFormat struct {
-	WhichVulns string
-	DateStr    string
+	ImageFlavorBranding string
+	WhichVulns          string
+	DateStr             string
+}
+
+type reportnoVulnsFoundEmailFormat struct {
+	ImageFlavorBranding string
 }
```

Comment:
Technically, you can reuse `reportEmailFormat` for formatting `noVulnsFoundEmailTemplate`. Just don't set `WhichVulns` and `DateStr`.

Reply:
Cool, I didn't know that worked. I'll do that then. While I'm at it I'll also rename `ImageFlavorbranding` as the name doesn't make much sense.